modules/odo-core-concepts.adoc

```diff
@@ -10,35 +10,20 @@
 `odo` abstracts Kubernetes concepts into terminology that is familiar to developers:
 
 Application::
-A typical application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that is used to perform a particular task.
+A typical application, developed using a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that performs a particular task.
```

Comment:
```suggestion
A typical application, developed using a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that does a particular task.
```
Per the ISG, `perform` should be avoided if possible.

Reply:
"perform verb
Do not use if more direct wording is possible. For example, instead of writing "Perform the product installation", write "Install the product"."

I think perform is ok here, because it needs some sort of verb like "execute", or "does" or whatever, so I don't think any of those are more direct in this instance.